symfony/plugins/orangehrmRESTPlugin/lib/Api/Leave/LeaveRequestAPI.php

```diff
@@ -223,6 +254,128 @@ public function getLeaveRequestPerEmployee()
 
     }
 
+    /**
+     * Get leave requests for accessible leave list employees for current request user
+     * @return Response
+     * @throws InvalidParamException
+     * @throws RecordNotFoundException
+     * @throws \ServiceException
+     */
+    public function getLeaveRequests()
```

Comment:
return type

Reply:
`symfony/plugins/orangehrmRESTPlugin/lib/Api/Leave/LeaveRequestAPI.php` this is already existing file with PHP 5.6 compatibility. And I didn't add types.